src/components/ImportSection.vue

```diff
@@ -156,13 +159,20 @@ export default {
 				const filePath = await picker.pick()
 				this.logger.debug(`Path "${filePath}" selected for import`)
 				if (!filePath.startsWith('/')) {
-					throw new Error()
+					throw new Error(`Invalid path: ${filePath}`)
+				}
+				// TODO remove the file extension check when the custom mime type filter is added
+				if (!filePath.endsWith('.nextcloud_export')) {
+					throw new Error(`Invalid file: ${filePath}, please choose a valid "*.nextcloud_export" file`)
 				}
 
 				try {
 					await confirmPassword()
 					this.startingImport = true
-					// TODO call API to start background job
+					await axios.post(generateOcsUrl('/apps/{appId}/api/v1/import', { appId: APP_ID }), {
```

Comment:
I assume we have CSRF checks in place for this ?

Reply:
I just checked. All routes have this unless disabled through an annotation, which we did not set.
all good